src/create_env/start9.rs

```diff
@@ -0,0 +1,163 @@
+                                "btcstandup://{}:{}@{}:8332/",
+                                name, info.password, tor_addr
+                            ),
+                            description: Some(format!("Quick Connect URL for {}", name)),
+                            copyable: true,
+                            qr: true,
+                        })
+                        .collect(),
+                    description: Some("Quick Connect URLs for each user".to_owned()),
+                    copyable: false,
+                    qr: false,
+                },),
+            },
+        )?;
+        Ok(cfg)
+    })
+    .await??;
+    let decorator = slog_term::TermDecorator::new().build();
+    let drain = slog_term::FullFormat::new(decorator).build().fuse();
+    let drain = slog_async::Async::new(drain).build().fuse();
+    let logger = slog::Logger::root(drain, slog::o!());
+    Ok(Env {
+        bind: ([0, 0, 0, 0], 8332).into(),
+        rpc_client: match cfg.bitcoind {
+            BitcoinCoreConfig::Internal {
+                address,
+                user,
+                password,
+            } => RpcClient::new(
+                AuthSource::Const {
+                    username: user,
+                    password,
+                },
+                format!("http://{}:8332/", address).parse()?,
+            )?,
+            BitcoinCoreConfig::External {
+                address,
+                user,
+                password,
+            } => RpcClient::new(
+                AuthSource::Const {
+                    username: user,
+                    password,
+                },
+                Uri::from_parts({
+                    let mut addr = address.into_parts();
+                    addr.scheme = Some(uri::Scheme::HTTP);
+                    addr.path_and_query = None;
+                    if let Some(ref auth) = addr.authority {
+                        if auth.port().is_none() {
+                            addr.authority = Some(format!("{}:8332", auth).parse()?);
+                        }
+                    }
+                    addr
+                })?,
+            )?,
+            BitcoinCoreConfig::QuickConnect { quick_connect_url } => {
+                let auth = quick_connect_url
+                    .authority()
+                    .ok_or_else(|| anyhow::anyhow!("invalid Quick Connect URL"))?;
+                let mut auth_split = auth.as_str().split(|c| c == ':' || c == '@');
+                let user = auth_split.next().map(|s| s.to_owned());
+                let password = auth_split.next().map(|s| s.to_owned());
+                RpcClient::new(
+                    AuthSource::from_config(user, password, None)?,
+                    format!(
+                        "http://{}:{}/",
+                        auth.host(),
+                        auth.port_u16().unwrap_or(8332)
+                    )
+                    .parse()?,
+                )?
+            }
+        },
```

Comment:
Can do, but I'm thinking maybe this code should be pulled out of the crate and moved to our wrapper [here](https://github.com/Start9Labs/btc-rpc-proxy-wrapper) anyway, importing the proxy as a lib, especially since the config it is referencing is defined there.